portfolio/src/main/webapp/index.html

```diff
@@ -98,5 +144,63 @@ <h1>Work Experience</h1>
           <h1>Contact Me</h1>
       </section>
     </div>
+
+    <!-- Lightbox Gallery -->
+          <div id="lightbox">
```

Comment:
Totally optional, but if you want a little extra work, try making it possible to close the lightbox with the escape key, in addition to the close link.

Reply:
Great idea! Added left and arrow key functionality as well while at it